src/Parser/Lexer/Source.idr

```diff
@@ -205,7 +193,9 @@ fromOctLit str
                   Nothing => 0 -- can't happen if the literal lexed correctly
                   Just n => cast n
 
-rawTokens : TokenMap SourceToken
+%hide Prelude.reverse
```

Comment:
Is this still necessary? Wouldn't it be better to qualify the references to `reverse` with their namespace, such as `List.reverse`?